lib/delta-sync/delta-file.js

```diff
@@ -80,12 +86,29 @@ export default class DeltaFile {
   async ingest(onFinishCallback) {
     console.log(`Start ingesting file ${this.id} stored at ${this.deltaFilePath}`);
     try {
+      //Couldn't we use data from a variable on this file object? Much faster than writing and immediately reading from the same file.
       const changeSets = await fs.readJson(this.deltaFilePath, {encoding: 'utf-8'});
+      console.log("WHILE INGESTING, this is a changeSets: ", JSON.stringify(changeSets));
       for (let {inserts, deletes} of changeSets) {
-        console.log(`Deleting data in all graphs`);
-        await deleteTriplesInAllGraphs(deletes);
+        //Filter on file triples
```

Comment:
I would like to have a big if/else block here. With the an environment to determine whether we need file sync. if We don't need filesync, I would fall back to the previous code, we know it works, as a safety measure. 
You can keep the change made on `insertTriplesInGraph`